File: WEEKLYCODE#2/compare_the_triplets/C++/compare_the_triplets.cpp
#include <iostream>
using namespace std;

int main() {
	int a[3],b[3],c1=0,c2=0,i;
	//taking inputs
	for(i=0;i<3;i++)
	    cin>>a[i];
	for(i=0;i<3;i++)
	    cin>>b[i];
	//calculating c1 & c2
	for(i=0;i<3;i++){
	    if(a[i]>b[i])
	        c1++;
	   if(b[i]>a[i])
	        c2++;
	}
	//printing results
	cout<<c1<<" "<<c2;

	return 0;
}

File: WEEKLYCODE#2/compare_the_triplets/Python/compare_the_triplets.py
#taking inputs
ar1=map(int,input().split())
ar2=map(int,input().split())
c1=0
c2=0
#calculating c1 & c2
for a,b in zip(ar1,ar2):
    if a>b:
        c1+=1
    if b>a:
        c2+=1
#printing results        
print(c1,c2)
